coral/src/app/features/approvals/acls/AclApprovals.test.tsx

```diff
@@ -376,6 +376,31 @@ describe("AclApprovals", () => {
       );
     });
 
+    it("filters by Request type", async () => {
```

Comment:
Do we want to add a test case that it applies the filter from search params?

Reply:
I think this is covered by this test? https://github.com/aiven/klaw/blob/main/coral/src/app/features/components/filters/RequestTypeFilter.test.tsx#L46

Reply:
But I guess we do miss some test cases that would be relevant in the same area of concern! Added them here: [c940b35](https://github.com/aiven/klaw/pull/933/commits/c940b356fcf8397c79da4e1eeb7db03cbcb4b056)